### PR TITLE
Adding a new notification and removing break-word in select buttons

### PR DESCRIPTION
**Pull request by marlex**
### Where

http://coverwallet.github.io/cw-components/basic/

* **Pivotal Story:** https://www.pivotaltracker.com/story/show/144496383
* **Rollbar Errors:** No.
* **Related PRs:** No.
### What

I new notification that will be used to give a long information. Also removed break words in the select buttons.

### How

screen shot 2017-05-17 at 15 29 51

screen shot 2017-05-17 at 11 23 42

### CODE REVIEW

File: lib/components/CwFlashNotification.jsx [hunk 1]
```diff
@@ -36,8 +36,9 @@ class CwFlashNotification extends React.Component {
         </aside>
         <div className="cw-flash-notification__content">
           {title && <h4 className="cw-flash-notification__title">{title}</h4>}
+          {title_small && <h5 className="cw-flash-notification__title-small">{title_small}</h5>}
           {subtitle && <h5 className="cw-flash-notification__description">{subtitle}</h5>}
-          <div className="cw-flash-notification__text">{this.props.children}</div>
+          <h5 className="cw-flash-notification__text">{this.props.children}</h5>
```
Comment (alejandroweb) [inline lib/components/CwFlashNotification.jsx]: Don't do this please, it is the container for child elements and using heading tags as container is not correct. If we want to add a button to the notification in the future it shoudn't be inside an `h5`.

File: lib/stylesheets/components/_cw_flash_notification.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .cw-flash-notification {
   @include display(flex);
-  min-height: 58px;
+  align-items: center;
```
Comment (alejandroweb) [inline lib/stylesheets/components/_cw_flash_notification.scss]: Use this mixin `@include align-items(center);` :D

File: lib/stylesheets/components/_cw_flash_notification.scss [hunk 3]
```diff
@@ -64,31 +66,31 @@
...
   &__controls {
-    padding: 0 17px;
+    padding: 0 15px;
   }
 
   &__close {
     display: inline-block;
     height: 100%;
     text-align: center;
-    padding-top: 18px;
```
Comment (alejandroweb) [inline lib/stylesheets/components/_cw_flash_notification.scss]: Something happened removing this line... look

image

icon should be 1px top to be aligned with text

File: examples/basic/app.js [hunk 4]
```diff
@@ -102,40 +102,48 @@ const App = function App() {
...
+          subtitle="Call (646) 844-9933 to purchase this policy."
           onClose={() => console.log('Custom close event handler.')}
         />
       </div>
 
       <div className="clearfix row">
         <CwFlashNotification
           type="info"
-          title="Your account need to be activated"
+          title_small="The start date of the original quote already passed. Please recalculate the quote in order to continue with the purchase."
```
Comment (dzmitry-kremez-itechart) [inline examples/basic/app.js]: use camelCase in js like `tittleSmall` or better `subTittle`

File: lib/components/CwFlashNotification.jsx [hunk 5]
```diff
@@ -36,6 +36,7 @@ class CwFlashNotification extends React.Component {
         </aside>
         <div className="cw-flash-notification__content">
           {title && <h4 className="cw-flash-notification__title">{title}</h4>}
+          {title_small && <h5 className="cw-flash-notification__title-small">{title_small}</h5>}
```
Comment (dzmitry-kremez-itechart) [inline lib/components/CwFlashNotification.jsx]: will we use `title_small` at the same time when we use `title`?

if not, then better to use BEM modificator like  `cw-flash-notification__title cw-flash-notification__title--small` in title class name

**Pull request merged**